.github/workflows/node.yml

```diff
@@ -1,17 +1,17 @@
-name: 'Tests: node.js >= 10'
+name: 'Tests: node.js 18+'
 
 on: [pull_request, push]
 
 jobs:
   tests:
     uses: ljharb/actions/.github/workflows/node.yml@main
     with:
-      range: '>= 10'
+      range: '^18.18.0 || ^20.9.0 || >=21.1.0'
```

Comment:
Node 22 is out, so maybe don't need to support Node 21, which is unstable?

Reply:
I've set `node.engines` to the same as ESLint is doing, but I'm happy to update if @ljharb wants to